public/js/index.js

```diff
@@ -0,0 +1,207 @@
+$(document).ready(() => {
+  let uploadedFiles = [];
+
+  $('#table-display-error').hide();
+  $('#upload-error').hide();
+  $('.file-index').hide();
+  $('.file-upload').fadeIn('slow');
+
+  // upload file button
+  $('#button-upload-file').click(() => {
+    $('#button-upload-file').removeAttr('disabled');
+    $('#select-file').empty();
+
+    if (uploadedFiles.length < 5) {
+      $('#button-upload-file').removeAttr('disabled');
+      $('#upload-error').hide();
+
+      uploadedFiles = $('#image-file')[0].files;
+      updateSelectOptions(uploadedFiles, '#select-file');
+      updateSelectOptions(uploadedFiles, '#select-file-search');
+    } else if (uploadedFiles.length === 0) {
```

Comment:
The following 2 `else` blocks seem to be the same. Is it possible to factor them into one?